sparql11/data-sparql11/functions/manifest.ttl

```diff
@@ -248,6 +274,16 @@
     mf:result  <length01.srx> ;
 	.
 
+:length01-non-bmp rdf:type mf:QueryEvaluationTest ;
+	mf:name    "STRLEN() on non-BMP unicode strings" ;
+	mf:feature sparql:strlen ;
+	dawgt:approval dawgt:Proposed ;
+    mf:action
```

Comment:
Spaces here, tab line above.

Reply:
This seems to the consistent throughout the entire manifest. I can normalize it file-wide if there's interest.

Reply:
Removed all tabs from the manifest.